postman/README.md

```diff
@@ -26,10 +26,11 @@ This directory contains the Postman collection for all Hyperswitch supported con
 
 - Make sure that you update the `tests` section where the necessary `javascript` code has to written/updated to test the feature (assertion checks where you verify the results obtained with the expected outcome)
 - If certain `tests` need to be run at the time of making a request, make sure you add them to the `Pre-request Script` section of the request
+- Make sure that the request body does not contain any comments else the `newman dir-export` command will fail which is used to export the collection to its directory structure
```

Comment:
Postman does allow using JSON comments in request bodies though:
https://blog.postman.com/introducing-support-for-json-comments-in-request-body/

Reply:
Yes, but the `dir-export` command fails. Might have to check with @knutties to fix it.